mx.truffleruby/suite.py

```diff
@@ -30,33 +30,33 @@
             "url": "https://opensource.org/licenses/EPL-1.0",
         },
         "BSD-simplified": {
-          "name": "Simplified BSD License (2-clause BSD license)",
-          "url": "http://opensource.org/licenses/BSD-2-Clause"
+            "name": "Simplified BSD License (2-clause BSD license)",
+            "url": "http://opensource.org/licenses/BSD-2-Clause"
         },
         "MIT": {
-          "name": "MIT License",
-          "url": "http://opensource.org/licenses/MIT"
+            "name": "MIT License",
+            "url": "http://opensource.org/licenses/MIT"
         },
         "Apache-2.0": {
-          "name": "Apache License 2.0",
-          "url": "https://opensource.org/licenses/Apache-2.0"
+            "name": "Apache License 2.0",
+            "url": "https://opensource.org/licenses/Apache-2.0"
         },
         "GPLv2": {
-          "name": "GNU General Public License, version 2",
-          "url": "http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html"
+            "name": "GNU General Public License, version 2",
+            "url": "http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html"
         },
         "zlib": {
-          "name": "The zlib License",
-          "url": "https://opensource.org/licenses/zlib"
+            "name": "The zlib License",
+            "url": "https://opensource.org/licenses/zlib"
         },
```

Comment:
It's added to unify the indentation in the file. And to align it with idea auto-formatter.